### PR TITLE
[CI-1283] Migrate to tool kit

### PR DESCRIPTION
**Pull request by AlbertoCuberoNavas**
[https://financialtimes.atlassian.net/browse/CI-1283](https://financialtimes.atlassian.net/browse/CI-1283)

### CODE REVIEW

**Review by GlynnPhillips — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,42 +1,63 @@
...
+		"stylelint-scss": "^4.3.0"
+	},
+	"scripts": {
+		"precommit": "secret-squirrel && npm run format && npm run fix",
+		"prepush": "npm run test",
+		"prepare": "npx snyk protect || npx snyk protect -d || true && husky install",
+		"preinstall": "[ \"$INIT_CWD\" != \"$PWD\" ] || npm_config_yes=true npx check-engine",
+		"build": "dotcom-tool-kit build:local",
+		"test": "dotcom-tool-kit test:local",
+		"start": "dotcom-tool-kit run:local",
```
Comment (juanSanchezAlcala) [inline package.json:39]: Maybe this script doesn't have much sense due we are not runing backend app

File: secret-squirrel.cjs [hunk 2]
```diff
@@ -1,8 +1,8 @@
 module.exports = {
 	files: {
 		allow: [
-			'.snyk',
-			'CODEOWNERS'
+			'.husky/pre-commit',
```
Comment (juanSanchezAlcala) [inline secret-squirrel.cjs:4]: we need to add old allowed files .snyk and CODEOWNERS

**Review by juanSanchezAlcala — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (juanSanchezAlcala) [reply package.json:39]: i saw that is required by toolkit

File: secret-squirrel.cjs (hunk 2 above)
Comment (AlbertoCuberoNavas) [reply secret-squirrel.cjs:4]: why?

Comment (juanSanchezAlcala) [reply secret-squirrel.cjs:4]: is not needed , isn't any issue that squirrel could raise

**Review by juanSanchezAlcala — APPROVED** [review]

**Pull request merged**